keyboards/hyper7/h7/v3/config.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+#define CONFIG_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xDEAD
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    yiancar
+#define PRODUCT         hyper7 v3.0
+#define DESCRIPTION     hyper7 v3.0 firmware compiled by bit-shifter
```

Comment:
```suggestion
```

Reply:
Removed - is there any need for the description field?